docs/mount.md

```diff
@@ -1,3 +1,7 @@
+<!--meta
+Documentation on m.mount(), which binds Mithril components to certain DOM nodes
```

Comment:
Not fond of the choice of "certain" here. Maybe "specific" or "external" would work, or even potentially removing it entirely (though I have concerns about it potentially misleading people).

```suggestion
Documentation on m.mount(), which binds Mithril components to external DOM nodes
```

```suggestion
Documentation on m.mount(), which binds Mithril components to specific DOM nodes
```

```suggestion
Documentation on m.mount(), which binds Mithril components to DOM nodes
```

Reply:
what about `Documentation on m.mount(), which binds a Mithril component to given DOM node`

Reply:
That works.